seccomp_unsupported.go

```diff
@@ -28,8 +28,8 @@ func LoadProfileFromBytes(body []byte, rs *specs.Spec) (*specs.LinuxSeccomp, err
 	return nil, fmt.Errorf("Seccomp not supported on this platform")
 }
 
-// Reload takes a Seccomp struct and a spec to reload the config
-func Reload(config *Seccomp, specgen *specs.Spec) error {
+// LoadProfileFromConfig takes a Seccomp struct and a spec to retrieve a LinuxSeccomp
+func LoadProfileFromConfig(config *Seccomp, specgen *specs.Spec) error {
```

Comment:
The function should return a tuple `(*specs.LinuxSeccomp, error)`

Reply:
Fixed as suggested.